src/transformers/visitors/toCircuitVisitor.ts

```diff
@@ -174,7 +174,7 @@ const visitor = {
 
       const newNode = buildNode('BinaryOperation', { operator });
       node._newASTPointer = newNode;
-      parent._newASTPointer[path.containerName] = newNode;
+      path.inList ? parent._newASTPointer.push(newNode) : parent._newASTPointer[path.containerName] = newNode;
```

Comment:
Fixed bug where sometimes nodes would be in an array of nodes rather than an object, so this line would fail